src/components/common/YourCompany/ChangeBusinessType.vue

```diff
@@ -18,7 +18,7 @@
         </span>
 
         <!-- Type change tooltip -->
-        <v-tooltip v-if="isFirm || isCoop"
+        <v-tooltip v-if="typeChangeInfo"
```

Comment:
This now checks whether the resource was actually declared (instead of assuming that firms and coops have the resource declared), and it removes some firm- and coop-specific code.